doc/rst/language/spec/procedures.rst

```diff
@@ -868,6 +997,20 @@ The ``const ref`` return intent is also available. It is a restricted
 form of the ``ref`` return intent. Calls to functions marked with the
 ``const ref`` return intent are not lvalue expressions.
 
+.. _Const_Return_Intent:
+
+The Const Return Intent
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``const`` return intent makes it up to the implementation to return a
+``const`` reference or to return a value.  When making this choice for a
+the implementation should make decisions consistent with ``const`` formal
+argument intents (see :ref:`The_Const_Intent`).
+
+Calls to functions marked with the ``const`` return intent are not lvalue
+expressions.
```

Comment:
I suggest dropping the "consistent" requirement as we do not have experience with it.

Also, merge the "lvalue" sentence into the previous paragraph.